assignment_1.py

```diff
@@ -0,0 +1,251 @@
+Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] on win32
```

Comment:
I guess you have copy pasted the code from terminal. Use VS Code to create and write python code. That's the correct way to do.